backend/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I believe `tslint` is deprecated now, I think the community use `eslint` instead.